_episodes/00-quickstart.md

```diff
@@ -0,0 +1,111 @@
+---
+title: "Quickstart Guide"
+teaching: 5
+exercises: 10
+questions:
+
+- "How do I activate the ESMValTool environment?"
+- "How do I configure the ESMValTool?"
+- "How do I run a recipe?"
+
+objectives:
+
+- "Activate the ESMValTool environment"
+- "Configure the ESMValTool"
+- "Successfully run a recipe"
+
+keypoints:
+- "See the installation guide for activating the ESMValtool environment."
+- "The config file has to be set up, uncommenting the environment you use."
+- "The command for running a recipe is `esmvaltool run`."
+---
+
+> ## Installing and getting started with the ESMValTool
+>
+> To install the ESMValTool please see 
+> [here](https://docs.esmvaltool.org/en/latest/quickstart/installation.html#)
+{: .callout}
+
+> ## Configuring ESMValtool
+>
+> With ESMValTool installed, we can now set up our configuration file (which 
+> tells ESMValTool how to run, where to get data from, and where to output it).
+>
+> You can find out more about this in the 
+> [documention](https://docs.esmvaltool.org/projects/ESMValCore/en/latest/quickstart/configure.html).
```

Comment:
Please link to the `configuration` episode instead, see https://esmvalgroup.github.io/ESMValTool_Tutorial/03-configuration/index.html